README.md

```diff
@@ -1,3 +1,124 @@
 # cloud-sql-go-connector
 *Warning*: This project is experimental, and is not an officially supported 
 Google product.
+
+The _Cloud SQL Go Connector_ provides IAM authorization to an app's connections 
```

Comment:
authorization for

Reply:
or better: provides IAM authorization and an encrypted connection for an app's connections to a Cloud SQL instance.

Reply:
In other words, should encryption be in here as one of the selling points?